aion/web/elm/src/Room/Notifications.elm

```diff
@@ -17,16 +17,39 @@ addToast toast ( model, cmd ) =
     Toasty.addToast myConfig ToastyMsg toast ( model, cmd )
 
 
+
+-- answer feedback toasts
+
+
 incorrectAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 incorrectAnswerToast =
-    addToast (Toasty.Defaults.Error "Error!" "Wrong Answer")
+    addToast (Toasty.Defaults.Error "Error!" "Wrong answer!")
 
 
 closeAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 closeAnswerToast =
-    addToast (Toasty.Defaults.Warning "Close one!" "Your Answer is Almost Correct")
+    addToast (Toasty.Defaults.Warning "Close one!" "Your answer is almost correct!")
 
 
 correctAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 correctAnswerToast =
-    addToast (Toasty.Defaults.Success "Good Answer!" "Your Answer is Correct")
+    addToast (Toasty.Defaults.Success "Good Answer!" "Your answer is correct!")
+
+
+
+-- question creation
+
+
+questionFormValidationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
```

Comment:
I think I've already answered to that above. Refactor can come later when our Panel section will be in more advanced stage, maybe we will be able to see more patterns to refactor. You know, _premature optimisation blah blah..._